src/currentToPower/index.js

```diff
@@ -8,8 +8,8 @@ const VOLTAGE = 230;
 const numberToFixedNumber = (number, digits) => Number(number.toFixed(digits));
 
 const paths = [
-  ...createFileList('output/current/idle'),
-  ...createFileList('output/current/idleWithBrowser'),
+  // ...createFileList('output/current/idle'),
+  // ...createFileList('output/current/idleWithBrowser'),
   ...createFileList('output/current/withGhostery'),
```

Comment:
Instead of creating empty output paths upfront, you can create subpaths on the fly in the `createFileList` helper method. Then, add to `.gitignore` only the root path of the output, and remove those `.gitkeep` files.

You can use the shell command `mkdir` (for example using shelljs package):

```
shelljs.mkdir('-p', resolve(outDir, path));
```

We are using this method in the build script in ghostery extension - https://github.com/ghostery/ghostery-extension/blob/mv3-release/extension-manifest-v3/scripts/build.js#L99

Reply:
This topic is for the next iteration. I'm thinking to take care of it here:
[Enhancements](https://github.com/ghostery/benchmarks/issues/4) :
- Measurement results in specific folders
- Delete profiles and measurements